### PR TITLE
[Cppcheck] Note for a json file containing the configuration of addons

### PR DESCRIPTION
**Pull request by suetsuguk**
See https://github.com/sider/runners/pull/1141

### CODE REVIEW

File: docs/tools/cplusplus/cppcheck.md [hunk 1]
```diff
@@ -141,3 +141,25 @@ linter:
 ```
 
 The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
+
+You can also specify a json file containing [the configuration of addons](http://cppcheck.sourceforge.net/manual.html#running-addons). For example specify the file path to the json file:
```
Comment (dev-yakuza) [inline docs/tools/cplusplus/cppcheck.md:145]: [suggestion] How about making simple?

```diff
- You can also specify a json file containing [the configuration of addons](http://cppcheck.sourceforge.net/manual.html#running-addons). For example specify the file path to the json file:
+ You can also specify a json file containing [the configuration of addons](http://cppcheck.sourceforge.net/manual.html#running-addons). For example:
```

Comment (ybiquitous) [reply docs/tools/cplusplus/cppcheck.md:145]: [suggestion] SourceForge is unstable, so I recommend using GitHub instead.

```suggestion
You can also specify a json file containing [the configuration of addons](https://github.com/danmar/cppcheck/blob/master/man/manual.md#running-addons). For example specify the file path to the json file:
```

GitHub is used also here:

https://github.com/sider/sider-docs/blob/d3c09f9df2029fe8f9bd4a0dbe96a3dcfdbeb40a/docs/tools/cplusplus/cppcheck.md#getting-started

File: docs/tools/cplusplus/cppcheck.md [hunk 2]
```diff
@@ -141,3 +141,25 @@ linter:
...
+
+```yaml
+linter:
+  cppcheck:
+    addon:
+      - config/misra.json
+```
+
+Note that the file path should be relative to the root directory where the sider.yml is located or the `root_dir` directory if it is specified.
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:154]: [nitpick]

```suggestion
Note that the file path should be relative to the root directory where the `sider.yml` is located or the `root_dir` directory if it is specified.
```

File: docs/tools/cplusplus/cppcheck.md (hunk 1 above)
Comment (suetsuguk) [reply docs/tools/cplusplus/cppcheck.md]: > [suggestion] How about making simple?
> 
> ```diff
> - You can also specify a json file containing [the configuration of addons](http://cppcheck.sourceforge.net/manual.html#running-addons). For example specify the file path to the json file:
> + You can also specify a json file containing [the configuration of addons](http://cppcheck.sourceforge.net/manual.html#running-addons). For example:
> ```

LGTM
Fixed. commit d4b3dff529d4654f794164245da0d92ecb2fbec7

File: docs/tools/cplusplus/cppcheck.md (hunk 2 above)
Comment (suetsuguk) [reply docs/tools/cplusplus/cppcheck.md]: Good catch.
Fixed. commit 778df532dff4afb8445115e871d72689a9d3977f and 94ea0c7777bcc713d46f8fc660c50cd1c2a2864b

**Pull request merged**